oat-sa/extension-tao-itemqti-pci
# [UNO-306] CUNO-315: PCI liquid - Map response not working

Related to : https://oat-sa.atlassian.net/browse/UNO-306
 
Do not allow `map response` response processing template for liquid interaction
  
#### How to test
 
- prepare an instance of TAO
- create an item
- try to add liquids interaction to it
- edit response of the added interaction and check that there is no `map response` option for response template
